src/transformers/no-duplicated-vue-import.ts

```diff
@@ -0,0 +1,108 @@
+import { LogContexts, LogLevels } from 'bs-logger'
+import type { ConfigSet } from 'ts-jest/dist/config/config-set';
+import type * as _ts from 'typescript'
+
+// this is a unique identifier for your transformer. `ts-jest` uses this for jest cache key
+export const name = 'no-duplicated-vue-import'
+// increment this each time you change the behavior of your transformer. `ts-jest` uses this for jest cache key
+export const version = 1
+const VUE_GLOBAL_NAME = 'vue'
+const ROOT_LEVEL_AST = 1
+
+/**
+ * @usage In jest config, define as
+ * // jest.config.js
+ * module.exports = {
+ *   // other configs
+ *   globals: {
+ *     'ts-jest': {
+ *       astTransformers: {
+ *         after: ['vue-ts-jest/dist/transformers/no-duplicated-vue-import']
+ *       }
+ *     }
+ *   }
+ * }
+ */
```

Comment:
wow, amazing. will the user need to duplicate this jest.config, or can we merge it in to their own jest config somehow?

Reply:
Actually it depends on how often users use es6+ in tsconfig.spec.json . 

If it is often the case, you can make a preset file and let users reference to that preset. The preset will contain this config.

If it is not often the case, users can opt-in/out this transformer.

Reply:
great. most of the users are not writing their own config anyway, but using a template. this will be good.